src/bulk/common.ts

```diff
@@ -640,7 +640,7 @@ function executeCommands(
     err.ok = 0;
     // Merge top level error and return
     mergeBatchResults(batch, bulkOperation.s.bulkResult, err, undefined);
-    callback();
```

Comment:
This seems like it was probably an oversight, why would we want to swallow the error here?